nle/nethack/nethack.py

```diff
@@ -45,24 +45,9 @@
     "pickup_burden:unencumbered",
 ]
 
-HACKDIR = os.getenv("HACKDIR")
-
-if HACKDIR is None:
-    # Somewhat HACKy way of getting HACKDIR from installed nethack.
-    script = shutil.which("nlehack")
-    if script is None:
-        raise FileNotFoundError("Didn't find nethack in path. Is it installed?")
-    with open(script, "r") as f:
-        while True:
-            line = f.readline()
-            if not line:
-                raise ValueError("Could not determine HACKDIR from installed nethack")
-            if line.startswith("HACKDIR="):
-                HACKDIR = line[8:-1]  # Exclude newline.
-                break
-
-
+HACKDIR = os.getenv("HACKDIR", os.path.expanduser("~/nethackdir"))
```

Comment:
We shouldn't make this assumption in in end, right?

Reply:
Indeed, I'm going to change this after I get the tests up and running :)

Reply:
Wait this is still in here?

Reply:
Sorry, I left this as-is since the changes for **robustly** installing `$HACKDIR` into the package require more work. Why is this such a problem right now? The end behaviour is mostly similar to `master`, since we set it [here](https://github.com/facebookresearch/nle/pull/38/files#diff-af3b638bc2a3e6c650974192a53c7291R13-R15)?

Reply:
No, this is actually worse than before. Before it was a directory in conda. How it interferes with the users NetHack version (potentially overriding save games on compilation).

Reply:
Okay, I've changed this to `sys.base_prefix/nethackdir`, which is basically what we had before.